test/browser/specs/update-snapshot.test.mjs

```diff
@@ -0,0 +1,49 @@
+import assert from 'node:assert'
+import fs from 'node:fs'
+import test from 'node:test'
+import { startVitest } from 'vitest/node'
+
+let vitest
+
+test.after(async () => {
+  // force exitCode mutated by vitest
+  process.exitCode = 0
+  await vitest?.close()
+})
+
+test('update snapshot', async () => {
```

Comment:
I thought this test might be too flaky, but CI somehow got through fine.

It might have some problems on Windows, so if that happens, then maybe we can introduce `{ skip: process.platform === "win32" }` here. Please feel free to re-run `test-browser-windows` job to do stress test.